action/remove.go

```diff
@@ -2,24 +2,70 @@ package action
 
 import (
 	"os"
+	"os/exec"
 	"path/filepath"
+	"strings"
 
 	"github.com/helm/helm/log"
+	"github.com/helm/helm/manifest"
 )
 
+// kubeGet executes the kubectl command, override in tests
+var kubeGet func(string) string
+
+func init() {
```

Comment:
Do you need an init block for this, or could you simply do the `= func...` part on line 14?